wormhole-attester/program/src/message.rs

```diff
@@ -44,7 +44,7 @@ impl<'a> Seeded<&P2WMessageDrvData> for P2WMessage<'a> {
             // See the note at 2022-09-05 above.
             // Change the version in the literal whenever you change the
             // price attestation data.
-            "p2w-message-v1".as_bytes().to_vec(),
```

Comment:
Are you sure you need to update this? Doesn't having batch_size as a seed already solve the problem?

Reply:
The format had changed and payload size had changed. The batch size refers to the number of symbols being attested, not the total payload size in bytes.

Reply:
Hey Stan, since this will create new accounts, can you make sure our accounts will remain topped up by a lot of pgas when we do the deployment on Pythnet?